packages/manager/apps/web/client/app/components/order-catalog-price/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "price_free": "Inclus",
+  "price_tax_excl_label": "{{ price }} HT",
```

Comment:
You forgot about the difference between `GST` and `VAT`

Reply:
GST / VAT on french translations ?